devops/actions/run-tests/e2e/action.yml

```diff
@@ -55,7 +55,7 @@ runs:
     if: inputs.testing_mode != 'run-only'
     shell: bash
     run: |
-      cmake -GNinja -B./build-e2e -S./llvm/sycl/test-e2e -DCMAKE_CXX_COMPILER="${{ inputs.cxx_compiler || '$(which clang++)'}}" -DLLVM_LIT="$PWD/llvm/llvm/utils/lit/lit.py" ${{ steps.cmake_opts.outputs.opts }}
+      cmake -GNinja -B./build-e2e -S./llvm/sycl/test-e2e -DCMAKE_CXX_COMPILER="${{ inputs.cxx_compiler || '$(which clang++)'}}" -DLLVM_LIT="$PWD/llvm/llvm/utils/lit/lit.py" -DCUDA_LIBS_DIR=/usr/local/cuda/lib64 -DCUDA_INCLUDE=/usr/local/cuda/include ${{ steps.cmake_opts.outputs.opts }}
```

Comment:
I might be overthinking, but have you tested this change on Windows? Since these paths are non-existent on Windows, will cmake/LIT throw any error or just ignore CUDA? If later, we should be good to proceed.

Reply:
We actually don't use this action on Windows, but I'll wait for the CI to pass to be extra safe.

Reply:
yep windows ci passed